src/measure_theory/function/conditional_expectation.lean

```diff
@@ -1795,6 +1796,30 @@ begin
   exact (condexp_add hf hg.neg).trans (eventually_eq.rfl.add (condexp_neg g)),
 end
 
+section real
+
+lemma radon_nikodym_deriv_ae_eq_condexp {f : α → ℝ} (hf : integrable f μ) :
+  @signed_measure.radon_nikodym_deriv α m
+    ((μ.with_densityᵥ f).trim hm) (μ.trim hm) =ᵐ[μ] μ[f | hm] :=
+begin
+  refine ae_eq_condexp_of_forall_set_integral_eq hm hf _ _ _,
+  { intros,
+    exact (integrable_of_integrable_trim hm (@signed_measure.integrable_radon_nikodym_deriv α m
+        ((μ.with_densityᵥ f).trim hm) (μ.trim hm))).integrable_on },
+  { intros s hs hlt,
+    conv_rhs { rw [← hf.with_densityᵥ_trim_eq_integral hm hs,
+                   ← @signed_measure.with_densityᵥ_radon_nikodym_deriv_eq α m
+                     ((μ.with_densityᵥ f).trim hm) (μ.trim hm) _
+                     (hf.with_densityᵥ_trim_absolutely_continuous hm)] },
+    rw [with_densityᵥ_apply (@signed_measure.integrable_radon_nikodym_deriv α m
+          ((μ.with_densityᵥ f).trim hm) (μ.trim hm)), ← set_integral_trim hm],
+    measurability },
+  { refine measurable.ae_measurable' _,
```

Comment:
Should I tag `measurable.ae_measurable'` with `measurability`?

Reply:
The measurability tactic is not aware of `ae_measurable'` at all for now, so no. `ae_measurable'` is currently not supposed to be used much, so is not very developed. But if we see that we need it after all we'll change that.